docs/diagrams/AddSequenceDiagram.puml

```diff
@@ -27,7 +27,7 @@ activate AddCommandParser
 AddCommandParser --> InTrackParser
 deactivate AddCommandParser
 
-InTrackParser -> AddCommandParser : parse("add n/Google p/SWE hp/65025300002 \ne/support@google.com a/16 Park Way t/Urgent")
+InTrackParser -> AddCommandParser : parse("add c/Google p/SWE \ne/support@google.com w/https://careers.google.com/ \n s/5000 Park Way t/Urgent")
```

Comment:
Same as above

Reply:
Thank you for the advice, it has been implemented.